skipper.go

```diff
@@ -1292,6 +1295,38 @@ func getRedisUpdaterFunc(namespace, name string, kdc *kubernetes.Client) func()
 	}
 }
 
+func updateEndpointsFromURL(address string) func() []string {
+	return func() []string {
+		/* #nosec */
+		resp, err := http.Get(address)
```

Comment:
likely you should start the function closure by assigning `address` to a local var like:
```
address := address
resp, err := http.Get(address)
... 
```

Reply:
tested and still fails, I saw `nosec` in couple of places

Reply:
if we don't fail then not